### PR TITLE
Adds dates to releases in the CHANGELOG

### PR DESCRIPTION
**Pull request by orta**
Would [be nice](https://twitter.com/soffes/status/685917811319816193) to see as an overview the dates for releases.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -95,7 +95,7 @@
 * Use Library/Caches rather than app support directory (Kornel Lesiński)
 * Check for a modal window being onscreen before trying to put up the Sparkle prompt (Alf Watt)
 * Fixed crashes on 10.7 (Chris Campbell, Ger Teunis)
-* Fixed Sparkle tags parsing (Tamás Lustyik)
+* Fixed Sparkle tags parsing (Tamás Lustyik)
```
Comment (orta) [inline CHANGELOG]: not sure what's happening here, I did all my editing in the GitHub editor, rather than a fancy text editor.

Comment (pornel) [inline CHANGELOG]: That's just unicode normalisation.

**Comment by pornel** [comment]
Thanks!

**Comment by orta** [comment]
Thank you, I'm [a fan](https://github.com/artsy/energy/pull/131#discussion_r49163050) of your work

**Comment by pornel** [comment]
:blush:

**Pull request merged**